filesystems/samba2/README.md

```diff
@@ -0,0 +1,14 @@
+# Introduction
+
+Samba is the standard Windows interoperability suite of programs for Linux and Unix.
+
+## Run Samba Docker Compose File.
+docker-compose up --build
```

Comment:
What is this building, I don't see a Dockerfile?

Reply:
It used the compose file only, so it used the `scitotec/test-samba-share` base image and added the configuration on top of it.